platform/pulp/platform/models/repository.py

```diff
@@ -1,10 +1,268 @@
 """
-Django models related to repositories.
+Repository related Django models.
 """
+from datetime import datetime
 
-from pulp.platform.models import Model
+from django.db import models
+from django.contrib.contenttypes import fields
 
+from pulp.platform.models import Model, Notes, Scratchpad, Config
 
-class Distributor(Model):
+
+class Content(Model):
     # placeholder
-    pass
+    type = models.TextField()
+    created = models.DateTimeField(auto_now_add=True)
+
+
+class Repository(Model):
+    """
+    Collection of content.
+
+    Fields:
+
+    :cvar name: The consumer name.
+    :type name: models.TextField
+
+    :cvar: description: An optional description.
+    :type: models.TextField
+
+    :cvar type: The repository type.
```

Comment:
What are the choices? How would someone discover those choices, besides asking on a PR? ;)

If this corresponds to the type on the Plugin model, maybe there should be a more concrete relationship?